.github/workflows/ci.yml

```diff
@@ -3,49 +3,59 @@ name: CI
 on: [push]
 
 jobs:
+  commitlint:
```

Comment:
So this is ok, but wouldn't we prefer to have this as a pre-commit hook? It seems like we'd want to check this as people are writing their commit messages rather than after the fact, which would mean annoying history re-writing